janusgraph-test/src/main/java/org/janusgraph/graphdb/configuration/validator/CompatibilityValidatorTest.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 JanusGraph Authors
```

Comment:
This file should in the folder`/janusgraph-test/src/test/java/org/janusgraph/graphdb/configuration/validator`. 
As you can see in the travis ci details, see for example here https://travis-ci.org/JanusGraph/janusgraph/jobs/487161233.

Reply:
In `main` are tests which should run against different backends or utilities and in `test` are actual tests which are independently from any backend. These tests are executed in Travis ci.

Reply:
Thanks for clarifying! Now I get it.